LayoutTests/imported/w3c/web-platform-tests/content-security-policy/gen/top.http-rp/worker-src-self/sharedworker-import.http-expected.txt

```diff
@@ -4,11 +4,11 @@ PASS Content Security Policy: Expects allowed for sharedworker-import to same-ht
 PASS Content Security Policy: Expects allowed for sharedworker-import to same-http origin and no-redirect redirection from http context.
 PASS Content Security Policy: Expects allowed for sharedworker-import to same-http origin and no-redirect redirection from http context.: securitypolicyviolation
 PASS Content Security Policy: Expects blocked for sharedworker-import to cross-http origin and keep-origin redirection from http context.
-PASS Content Security Policy: Expects blocked for sharedworker-import to cross-http origin and keep-origin redirection from http context.: securitypolicyviolation
+FAIL Content Security Policy: Expects blocked for sharedworker-import to cross-http origin and keep-origin redirection from http context.: securitypolicyviolation assert_equals: One violation event should be fired expected 1 but got 0
```

Comment:
This seems like a regression??

Reply:
I'm uploading a revised patch for EWS, but will investigate this and resolve in a follow-up.